src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -1,25 +1,24 @@
 <template id="limited-restoration-panel">
   <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
-    <v-radio id="twentyFour-radio" name="selectMonths" label="Two Years" :value=24 />
-    <v-radio id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
-    <v-radio id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
-    <v-radio id="six-radio" name="selectMonths" label="6 months" :value=6 />
-    <v-row class="ml-0 mt-0">
-      <v-radio id="custom-months" name="selectMonths" value="customMonths" />
-      <v-text-field
-        class="shrink"
-        hide-details="auto"
-        type="number"
-        dense
-        hide-spin-buttons
-        min=1
-        max=24
-        :rules="monthRules"
-        v-model="numberOfMonths"
-        :disabled="!customMonths"
-        @change="onMonthsChanged"
-        filled
-      />
+    <v-radio class="radio-button pt-2" id="twentyFour-radio" name="selectMonths" label="2 years" :value=24 />
+    <v-radio class="radio-button pt-2" id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
+    <v-radio class="radio-button pt-2" id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
+    <v-radio class="radio-button pt-2" id="six-radio" name="selectMonths" label="6 months" :value=6 />
+    <v-row class="ml-0 mt-0 radio-button pt-2">
+      <v-radio class="mt-n4" id="custom-months" name="selectMonths" value="customMonths" />
```

Comment:
Could "mt-n4" be moved to line 7?

Reply:
No, this is done to center only the radio bubble with the text field. I showed it to Ethan yesterday and that's how the UX team wants it!

Reply:
Without the "mt-n4" in v-radio:
![without](https://user-images.githubusercontent.com/122301442/222504980-964d53ce-4674-4f1e-8bd3-0ac9d0e69945.PNG)
With the "mt-n4" in v-radio:
![with](https://user-images.githubusercontent.com/122301442/222505085-5db5f82a-5b4a-42dd-b2b8-b70ed665bcac.PNG)

Reply:
Ah, OK.